### ISSUE TITLE
First tab doesn't load data from API

### ISSUE THREAD

**Issue by AliaMYH** [react-native-community/react-native-tab-view#515]
### Current behaviour

Currently, I have an API being called to load data for lists in the tab view, but the call is placed in _onIndexChange(). So the first page of the tab view is not loaded unless i swipe away from it and then back to it again ( ie an index change that leads to it). 

I need to be able to load the page automatically without having to swipe away and then back to it. 

### Code sample

```
_handleIndexChange = index => {

    this.props.setCorrectIndex(null);
    this.props.productsIsLoading(true);
    this.setState({ index });

    this.props.subcategoryChosen(this.state.routes[index].key);

    var alreadyLoaded = false;
    console.log('checking index in pager ');
    for ( var i in this.props.products){
        if ( this.props.products[i].sub_id === this.state.routes[index].key){
          console.log('index ' + index + 'is already loaded')
          alreadyLoaded = true;
          this.props.setCorrectIndex(i);
          this.props.productsIsLoading(false);
          break;
        }

    }
    if ( alreadyLoaded === false)
    {
      console.log('index ' + index + 'is not loaded, so fetching from api')

      this.props.getProducts(this.state.routes[index].key);
    }

    console.log('on index change key: ' + this.state.routes[index].key + ' index is : '+ index);

  }
```

The point here is the get products call. I need that to be called on the first index automatically without index changes. 

### What have you tried

Can I add a componentWillMount() function and place the call there? Or shall I place the call in my constructor right after initializing the index? I'm not sure what to do. 

### Your Environment

| software              | version
| --------------------- | -------
| ios or android        |android 5
| react-native          | react native 5.x
| react-native-tab-view | 0.0.74

**Comment by prashantvv** [comment]
Same issue happening with me. I think calling the api from previous component & then pass data to new component where tabview is added will may work. I also tried to force update after api call, but it just didn't work. So the fastest option is to call the api on previous page

**Comment by satya164** [comment]
Hey, I just released a new alpha `2.0.0-alpha.0` of the library. It's rewritten using [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler) and [react-native-reanimated](https://github.com/kmagiera/react-native-reanimated) addresses a many platform specific bugs and performance problems. The documentation is updated as well.

Please try the new version and see if it addresses your issue. If not, please open a new issue following the issue template.
